apps/web/src/lib/components/ui/community-details/AgentDetails.svelte

```diff
@@ -0,0 +1,68 @@
+<script lang="ts">
+	import dayjs from 'dayjs';
+	import relativeTime from 'dayjs/plugin/relativeTime';
+	import type { Agent } from '$lib/types/models';
+
+	dayjs.extend(relativeTime);
+
+	export let AgentDisplayDetails: Agent;
+
+
+	function timeSince(dateIsoString: Date | string | number) {
+		return dayjs(dateIsoString).fromNow(true);
+	}
+</script>
+
+<div class="mb-8 flex flex-col items-center justify-center">
+	<div class="relative">
+		<img
+			src={AgentDisplayDetails?.avatar}
```

Comment:
Take some time to learn about `null` and `undefined` types in TypeScript, very important for good code!
This article might help: https://www.w3schools.com/typescript/typescript_null.php
Understand what the `?.` is used for and how to use it properly. 
If help needed on this, let me know! 👍

Reply:
okay i will , i thought the agentDisplay might return undefined that why i used ?.